### PR TITLE
Add BPF recorder docs

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:
Adding docs to the installation and usage guide.
#### Which issue(s) this PR fixes:

None
#### Does this PR have test?
None

#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: installation-usage.md [hunk 1]
```diff
@@ -495,6 +496,112 @@ test-recording-nginx   Installed   15s
...
+I1115 12:02:48.406205  110307 bpfrecorder.go:373] bpf-recorder "msg"="Getting syscalls map"
+I1115 12:02:48.406287  110307 bpfrecorder.go:379] bpf-recorder "msg"="Getting comms map"
+I1115 12:02:48.406862  110307 bpfrecorder.go:396] bpf-recorder "msg"="Module successfully loaded, watching for events"
+I1115 12:02:48.406908  110307 bpfrecorder.go:677] bpf-recorder "msg"="Unloading bpf module"
+I1115 12:02:48.411636  110307 bpfrecorder.go:176] bpf-recorder "msg"="Starting GRPC API server"
+```
+
+The recorder does a system sanity check on startup to ensure everything works as
+expected. This includes a `load` and `unload` of the BPF module. If this fails,
+please open an issue find out what went wrong.
```
Comment (jhrozek) [inline installation-usage.md:538]: open an issue so that we can find out?

**Review by jhrozek — COMMENTED** [review]
One grammar nit, otherwise looks good.

File: installation-usage.md (hunk 1 above)
Comment (saschagrunert) [reply installation-usage.md]: Fixed 👍

**Pull request merged**